mpisppy/generic_cylinders.py

```diff
@@ -128,7 +128,19 @@ def _name_lists(module, cfg, bundle_wrapper=None):
 
 
 #==========
-def _do_decomp(module, cfg, scenario_creator, scenario_creator_kwargs, scenario_denouement, bundle_wrapper=None):
+def do_decomp(module, cfg, scenario_creator, scenario_creator_kwargs, scenario_denouement, bundle_wrapper=None):
+    """Essentially, the main program for decomposition
+
+    Args:
+       module (Python module): the model file with required functions
```

Comment:
```suggestion
       module (Python module or class): the model file with required functions or a class with the required methods.
```

I think this is correct, and gives the user more flexibility in interfacing with mpi-sppy.